CMakeLists.txt

```diff
@@ -264,6 +264,12 @@ else(MSVC)
   endif()
 endif(MSVC)
 
+# USE_MICRO is on by default, but it is not needed for Hexagon builds nor will
+# it function properly with USE_RPC disabled.
+if(USE_HEXAGON OR NOT USE_RPC)
+  set(USE_MICRO OFF)
```

Comment:
I suggest for this case we just show an error and say this mode is not supported. changing the user specified configs silently is not ideal.

Reply:
Thanks, updated PR to remove this until the best way of doing so is determined.